### ISSUE TITLE
自制数据集能通过tools/misc/browse_dataset.py  正确绘制3d框，是不是能证明数据集标签是正确的？

### ISSUE THREAD

**Issue by sunbin1993** [open-mmlab/mmdetection3d#2545]
自制数据集没有标定文件和图像，使用create_dataset.py创建pkl时，修改kitti_dataset.py，将np.linalg.inv(lidar2cam))改为np.identity(4))
        gt_bboxes_3d = CameraInstance3DBoxes(
            ann_info['gt_bboxes_3d']).convert_to(self.box_mode_3d,
                                                 np.identity(4))
                                                 #np.linalg.inv(lidar2cam))

**Comment by JingweiZhang12** [comment]
@sunbin1993 Yes, the `browse_dataset.py` will show the gt_bboxes in the dataset. If the boxes in the visualization are in the suitable location, the labels are right.

**Comment by sunbin1993** [comment]
> @sunbin1993 Yes, the `browse_dataset.py` will show the gt_bboxes in the dataset. If the boxes in the visualization are in the suitable location, the labels are right.

好的，多谢回复
